src/hoc/DragDropHandler.tsx

```diff
@@ -53,9 +53,7 @@ export const DragDropHandler = ({
 
   const [{ isDragging }, drag, preview] = useDrag({
     type: accept,
-    item: () => {
-      return { task: task }
-    },
+    item: { task: task },
```

Comment:
This can be simplified. But I think we should provide dependency array here here with task as input.